front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -35,7 +35,7 @@ type StudyParams = {
   projectId: string;
 };
 
-const emptyStudy: StudyForm = { name: '', tags: [] };
+const emptyStudy: StudyForm = { name: '', state: studyStates[0], tags: [] };
```

Comment:
I'm annoyed by this line. #5714 explicitly specifies `Starting` should be the default value. However, [`consts.ts`](https://github.com/osrd-project/osrd/blob/8125e629e6fb17f9a1f972599915f96823bf0606/front/src/applications/operationalStudies/consts.ts#L132-L136) doesn't export `STUDY_STATES` that would allow me to use `STUDY_STATES.started`.

Reply:
Yes I agree with you, you can export STUDY_STATES and use STUDY_STATES.STARTING (can you also rewrite all the const in capital letters please ?) :)